storagetransfer/quickstart.py

```diff
@@ -26,8 +26,8 @@
 from google.cloud import storage_transfer
 
 
-def create_one_time_transfer(project_id: str, source_bucket: str,
-                             sink_bucket: str):
+def create_one_time_transfer(project_id: str = "my_project_id", source_bucket: str = "my_source_bucket",
+                             sink_bucket: str = "my_sink_bucket"):
     """Creates a one-time transfer job."""
```

Comment:
Can we move the comments explaining each argument to this docstring (similarly to this [example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/b1d9789da21c13699ca3eb2806946f29f8374a90/compute/client_library/snippets/instances/create.py#L60))?